code/game/antagonist/station/traitor.dm

```diff
@@ -101,9 +101,9 @@ var/datum/antagonist/traitor/traitors
 	traitor_mob.faction = "syndicate"
 	spawn_uplink(traitor_mob)
 	// Tell them about people they might want to contact.
-	var/mob/living/carbon/human/M = get_nt_opposed()
+	var/mob/living/carbon/human/M = get_poor()
```

Comment:
get_poor() is not an appropriate name for a function here. Should be `get_economic_status`.

Reply:
Disagree, this proc is only used to get people of low status.

Reply:
True.